static/js/settings_org.js

```diff
@@ -868,6 +868,31 @@ function _set_up() {
     }
     realm_icon.build_realm_icon_widget(upload_realm_icon);
 
+    $('#deactivate_realm_button').on('click', function (e) {
+        if (!overlays.is_modal_open()) {
+            e.preventDefault();
+            e.stopPropagation();
+            overlays.open_modal('deactivate-realm-modal');
+        }
+    });
+
+    $('#do_deactivate_realm_button').on('click', function () {
+        if (overlays.is_modal_open()) {
+            overlays.close_active_modal();
+        }
+        channel.del({
+            url:'/json/realm/deactivate_realm',
+            success: function () {
+                window.location.href = "/accounts/deactivated/";
```

Comment:
Do we need this?  I think something like this would be better done in an event handler in `server_events_dispatch.js` for the event you added (since we want this to occur for all users, not just the target one).